tests/acceptance/docker/documentServer/config/onlyoffice/entrypoint-override.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+# we can't mount it directly because the run-document-server.sh script wants to move it
+cp /etc/onlyoffice/documentserver/local.dist.json /etc/onlyoffice/documentserver/local.json
+
+/app/ds/run-document-server.sh
```

Comment:
instead of an separate file may be add directly this to the only office service? possible?